src/shared/lib/MutationUtils.tsx

```diff
@@ -51,172 +56,51 @@ export const SELECTOR_VALUE_WITH_VUS = [
-                        ],
-                }}
-            >
-                Inframe
-            </strong>
-        ),
-        [ProteinImpactType.INFRAME_UNKNOWN_SIGNIFICANCE]: (
-            <strong
-                style={{
-                    color:
-                        proteinImpactTypeColorMap[
-                            ProteinImpactType.INFRAME_UNKNOWN_SIGNIFICANCE
-                        ],
-                }}
-            >
-                Inframe
-            </strong>
-        ),
-        [ProteinImpactType.SPLICE_PUTATIVE_DRIVER]: (
-            <strong
-                style={{
-                    color:
-                        proteinImpactTypeColorMap[
-                            ProteinImpactType.SPLICE_PUTATIVE_DRIVER
-                        ],
-                }}
-            >
-                Splice
-            </strong>
-        ),
-        [ProteinImpactType.SPLICE_UNKNOWN_SIGNIFICANCE]: (
-            <strong
-                style={{
-                    color:
-                        proteinImpactTypeColorMap[
-                            ProteinImpactType.SPLICE_UNKNOWN_SIGNIFICANCE
-                        ],
-                }}
-            >
-                Splice
-            </strong>
-        ),
-        [ProteinImpactType.FUSION_PUTATIVE_DRIVER]: (
-            <strong
-                style={{
-                    color:
-                        proteinImpactTypeColorMap[
-                            ProteinImpactType.FUSION_PUTATIVE_DRIVER
-                        ],
-                }}
-            >
-                SV/Fusion
-            </strong>
-        ),
-        [ProteinImpactType.FUSION_UNKNOWN_SIGNIFICANCE]: (
-            <strong
-                style={{
-                    color:
-                        proteinImpactTypeColorMap[
-                            ProteinImpactType.FUSION_UNKNOWN_SIGNIFICANCE
-                        ],
-                }}
-            >
-                SV/Fusion
-            </strong>
-        ),
-        [ProteinImpactType.OTHER]: (
-            <strong
-                style={{
-                    color: proteinImpactTypeColorMap[ProteinImpactType.OTHER],
-                }}
-            >
-                Other
-            </strong>
-        ),
-        [DriverVsVusType.DRIVER]: (
-            <strong
-                style={{
-                    color: proteinImpactTypeColorMap[DriverVsVusType.DRIVER],
-                }}
-            >
-                Driver
-            </strong>
-        ),
-        [DriverVsVusType.VUS]: (
-            <strong
-                style={{
-                    color: proteinImpactTypeColorMap[DriverVsVusType.VUS],
-                }}
-            >
-                VUS
-            </strong>
-        ),
-    };
+    let ret = [
+        ...SELECTOR_VALUE_WITH_VUS,
+        DriverVsVusType.DRIVER,
+        DriverVsVusType.VUS,
+    ].reduce(
+        (obj, cur) => ({
+            ...obj,
```

Comment:
neglibile in this case but this is kind of wasteful (copying values onto new object in every iteration.   what i would do her is use lodash:
```
_([...]).keyBy().mapValues(your jsx function)
```